assets/src/dashboard/app/reducer/media.js

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { reshapePublisherLogo } from '../serializers';
+
+export const ACTION_TYPES = {
+  LOADING_MEDIA: 'loading_media',
+  ADD_MEDIA_SUCCESS: 'add_media_success',
+  ADD_MEDIA_FAILURE: 'add_media_failure',
+  FETCH_MEDIA_SUCCESS: 'fetch_media_success',
+  FETCH_MEDIA_FAILURE: 'fetch_media_failure',
+};
+
+export const defaultMediaState = {
+  error: {},
+  isLoading: false,
+  uploadedMediaIds: [],
+  publisherLogos: {},
```

Comment:
I'm meh about this naming. Technically it's just media, but it's only getting used for publisherLogos and this seems easier to read to me.

Reply:
Figured out some better names :) Keeping purpose of media agnostic in the media reducer (which was bothering me). Update is incoming.